lms/djangoapps/grades/models.py

```diff
@@ -517,7 +517,7 @@ class Meta(object):
 
     # Information about the course grade itself
     percent_grade = models.FloatField(blank=False)
-    letter_grade = models.CharField(u'Letter grade for course', blank=False, max_length=255)
+    letter_grade = models.CharField(u'Letter grade for course', blank=False, null=False, max_length=255)
```

Comment:
I thought you were going to set blank=True as well.  Is there a reason you didn't?